src/Microsoft.DotNet.Cli.Utils/Telemetry.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+using Microsoft.Extensions.PlatformAbstractions;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class Telemetry
```

Comment:
Can you add an ITelemetry interface with the TrackCommand method and then use that in the dotnet Program? Then you can write a unit test that verifies that TrackCommand is being called with the appropriate parameters.